test/integration/application.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Application', function() {
+  let app;
+  let fooView;
+  let barView;
+
+  beforeEach(function() {
+    cy.server();
+
+    cy.visitMn(function({_, Backbone, Marionette}) {
+      const Application = Marionette.Application.extend({
+        region: '#app-hook',
+      });
+
+      fooView = new Marionette.View({
+        template: _.template('Application region content')
+      });
+      barView = new Marionette.View({
+        template: _.template('<h1 class="title">Application region content</h1>')
+      });
+
+      app = new Application();
+      app.start();
+    });
+  });
+
+  it('should show a view without additional arguments', function() {
+    app.showView(fooView);
```

Comment:
could be something like `FooView = Marionette.View;`  then in the tests do `new FooView({`?  I dunno.